src/app/effect.rs

```diff
@@ -1,18 +1,18 @@
 use super::{MessageMapper, Notification};
 
-pub enum Effect<Ms, GMs> {
+pub(crate) enum Effect<Ms> {
     Msg(Option<Ms>),
-    GMsg(GMs),
     Notification(Notification),
     TriggeredHandler(Box<dyn FnOnce() -> Option<Ms>>),
 }
 
-impl<Ms: 'static, OtherMs: 'static, GMs> MessageMapper<Ms, OtherMs> for Effect<Ms, GMs> {
-    type SelfWithOtherMs = Effect<OtherMs, GMs>;
-    fn map_msg(self, f: impl FnOnce(Ms) -> OtherMs + 'static + Clone) -> Effect<OtherMs, GMs> {
+impl<Ms: 'static, OtherMs: 'static> MessageMapper<Ms, OtherMs> for Effect<Ms> {
+    type SelfWithOtherMs = Effect<OtherMs>;
+    fn map_msg(self, f: impl FnOnce(Ms) -> OtherMs + 'static + Clone) -> Effect<OtherMs> {
         match self {
             Effect::Msg(msg) => Effect::Msg(msg.map(f)),
-            Effect::GMsg(g_msg) => Effect::GMsg(g_msg),
+            // @TODO_B: Remove. App has a few seemingly important pub fn's that depend on this,
+            // need to understand how to change them to allow for its removal.
```

Comment:
I've add `Effect::Notification` recently, I assume we don't want to remove it.

Reply:
Yeah, my bad. Classic human error: removed the field without removing the comment, causing confusion [here](https://github.com/seed-rs/seed/blob/608631a2c7a1075e24e6345bfbf79ae86d7aaf11/src/app/effect.rs#L16)